nearcore/src/config.rs

```diff
@@ -1160,10 +1161,19 @@ async fn download_file_impl(
     let https_connector = hyper_tls::HttpsConnector::new();
     let client = hyper::Client::builder().build::<_, hyper::Body>(https_connector);
     let mut resp = client.get(uri).await?;
+    let bar_opt = resp.size_hint().upper().map(|file_size| {
+        let bar = ProgressBar::new(file_size);
+        bar.set_style(ProgressStyle::default_bar()
+            .template("{spinner:.green} [{elapsed_precise}] [{wide_bar:.cyan/blue}] {bytes}/{total_bytes} ({eta})")
+            .progress_chars("#>-"));
+        bar
+    });
```

Comment:
GitHub ate my comment?

```suggestion
    }).unwrap_or_else(ProgressBar::hidden);
```

to avoid dealing with options down the line.

Reply:
Even better, use a style which would shows how much has been downloaded and `per_sec`.

Reply:
Nice idea!